### DISCUSSION TITLE
AI 應用廣泛使用水量飆升，五年恐成長百萬倍

### DISCUSSION THREAD

**Discussion by Patsy17** [1101-GS4719-C/Python-Community-GS4719-C#1821] [General]
隨著應用AI的需求逐漸增加，應用越廣泛，所需要使用的水來讓資料中心散熱就會越來越多，
在2027年全球 AI 應用需求可能消耗掉 66 億立方公尺水資源，幾乎是美國華盛頓州全年用水量。
未來能源與水資源都將會是嚴重的問題，特別是一些有乾旱問題的地區！

資料來源：https://technews.tw/2023/12/27/widespread-use-of-water-for-ai-applications-soars/

**Comment by eric202501** [comment]
乾旱問題嚴重的地區，這樣的趨勢可能會帶來更大的影響，進一步加劇水資源短缺的問題。因此，在發展大AI應用的同時，我們也需要思考如何妥善使用水資源，希望未來能發現更節約的冷卻系統。

**Comment by Ryanlolry** [comment]
AI應用的廣泛發展帶來了日益增長的水資源需求，特別是用於散熱的冷卻作業。根據預測，在2027年，全球AI應用可能消耗掉66億立方公尺的水資源，幾乎等同於美國華盛頓州全年的用水量。這將使未來能源和水資源面臨嚴峻挑戰，尤其是在一些乾旱問題嚴重的地區。這項數據突顯了AI技術發展所帶來的可持續性挑戰，需要針對能源和水資源的合理利用提出解決方案，以確保AI的持續發展不會對環境造成嚴重影響。

**Comment by chuanmin0317** [comment]
AI的大規模應用通常需要大量的計算資源，這導致資料中心的運作需要更多的能源，同時也需要水來進行冷卻。這樣的趨勢對於全球能源和水資源的需求都帶來了巨大的壓力，未來可能需要研發更加先進的冷卻方法來解決水資源不足的問題

**Comment by FakeWood** [comment]
有點好奇這些硬體是怎麼冷卻的，水居然是耗材，
我以為水冷系統都是封閉的，難道是給伺服器們吹水冷扇?
會不會哪天有某個神奇公司學核電廠，用海水來冷卻呢?

**Comment by YANRONG0123** [comment]
之前有提過在南部設置科技廠房，但系上老師就覺得不太可能，畢竟南部有段時間缺水相當嚴重，所以要思考的地方不會是根據氣候而決定，而是要利用周邊的資源來面對如此龐大的需求，例如使用海水淡化廠，雖然成本高，但至少是穩定的供應水源。
